### DISCUSSION TITLE
Sakura docker 使用问题

### DISCUSSION THREAD

**Discussion by Heroleaves** [natfrp/wiki#399] [使用问题] [answered]
https://doc.natfrp.com/app/ugreen.html 跟着教程，最后启动报错
![image](https://github.com/natfrp/wiki/assets/92717245/c79490c4-45df-4713-a2bd-145340a906c3)

**Comment by Heroleaves** [comment]
我是绿联dx4600

**Comment by fakeboboliu** [comment, marked as answer]
这是因为国内某些 docker 加速器缓存了我们多年前有问题的版本，并将那个版本作为它认为的最新版本向希望得到最新版本的用户提供。
此加速器同样会造成其他知名或不知名镜像出现问题，建议您不要使用。

您可以根据 [此指南](https://gist.github.com/y0ngb1n/7e8f16af3242c7815e7ca2f0833d3ea6) 替换其他可用加速器，
或者在进行安装的第一步时通过将 `natfrp/frpc` 替换为 `natfrp/frpc:版本`（比如 `natfrp/frpc:0.51.0-sakura-2`）来强制获取指定版本。
